modules/ding_frontend/ding_frontend.strongarm.inc

```diff
@@ -44,7 +44,7 @@ function ding_frontend_strongarm() {
   $strongarm->name = 'dynamic_background_css';
   $strongarm->value = array(
     'selector' => 'body > #page.ding2-site-template',
-    'css' => 'background-size: cover;
+    'css' => 'background-size: cover;
```

Comment:
What is changed here?

Reply:
Possibly it was changed new line symbol at the end of the line and git is not showing this.